documentation/sphinx/source/tls.rst

```diff
@@ -138,12 +138,14 @@ Default Peer Verification
 The default peer verification is ``Check.Valid=1``.
 
 Default Password
-^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^
 
 There is no default password. If no password is specified, it is assumed that the private key is unencrypted.
 
-Parameters and client bindings
-------------------------------
+Permissions
+-----------
+
+All files used by TLS must have sufficient read permissions such that the user running the FoundationDB server or client process can access them. It may also be necessary to have similar read permissions on the parent directories of the files used in the TLS configuration.
```

Comment:
Does it have a reason for asking read permission on the parent dir?

Reply:
I'm not really sure to be honest, but see https://forums.foundationdb.org/t/facing-issue-with-tls-setup/1463/2.